src/ICeWall/doc/add_cell.md

```diff
@@ -0,0 +1,29 @@
+## ICeWall add_cell
+### Synopsis
+```
+  % ICeWall add_cell [-name <name>] \
```

Comment:
This is essentially using the ICeWall command as a namepace for the commands.
It makes it different from every other part of openroad that uses commands in the global namespace. I would expect this to be a command named "place_pad" to be somewhat consistent with "global_placement" and "detailed_placement". No ICeWall prefix. No "add_cell" sub-command.

Reply:
TCL commands moved to the global namespace - no ICeWall prefix

Reply:
Done - all commands now in the global namespace